src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(ooo => ooo.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
```

Comment:
```suggestion
          {tabs.map({ id, title, ..... }=> (
```

![image](https://github.com/mate-academy/react_tabs-js/assets/70009205/bf071b0f-2ab3-4803-ba82-5aadefaa6446)